docs/index.md

```
@@ -18,0 +1,180 @@
+      <h3>Request Preservation</h3>
+      <p>Ensures all incoming requests are processed even during scale operations</p>
+    </div>
+  </div>
+</div>
+
+<!-- How It Works Section -->
+<div class="how-it-works-section">
+  <h2>How It Works</h2>
+  <div class="how-it-works-content">
+    <div class="how-it-works-image">
+      <img src="images/modes.png" alt="Diagram illustrating KubeElasti proxy and serve modes">
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)